samples/subsys/pm/power_state/README.rst

```diff
@@ -0,0 +1,37 @@
+.. _power_state:
```

Comment:
Move this to samples/boards/stm32/power_mgmt and make it H7 specific.

Reply:
done